modules/images-configuration-create-config-map.adoc

```diff
@@ -0,0 +1,61 @@
+[id="steps-for-manual-configmap-creation"]
+= Steps to create the image signature ConfigMap
+
+The following commands will create and apply the image signature ConfigMap to the cluster you want to update. 
+
+.Procedure
+
+. Find the link:https://access.redhat.com/solutions/4583231[version] you want the cluster to update.
+
+. Add the version to OCP_RELEASE_NUMBER environment variable.
++
+----
+$ OCP_RELEASE_NUMBER=<{product-version}.0>
```

Comment:
This is implying that product-version should only be set to major.minor? If so we should explain that and perhaps even give an example. I just see it not being clear to some customers and resulting in support cases.

Reply:
This will render as 4.5.0 for 4.5 docs, etc., and my impression is that is intended to be enough of an example for users to replace with 4.5.9, or whatever version they actually want to mirror.  Maybe a `For example:` at the end of the above paragraph would make that clearer?

Reply:
NM. It's been a while since I've looked at at our docs and I totally forgot about the macros.